app/forms/claim/mentor_training_form.rb

```diff
@@ -0,0 +1,81 @@
+class Claim::MentorTrainingForm < ApplicationForm
+  include Rails.application.routes.url_helpers
+  attr_accessor :claim, :mentor_training, :hours_completed, :custom_hours_completed
+
+  validate :hours_completed_presence
+  validate :validate_mentor_training, unless: -> { errors.any? }
+
+  delegate :mentor_full_name, to: :mentor_training
+
+  def persist
+    updated_mentor_training.save!
+  end
+
+  def to_model
+    mentor_training
+  end
+
+  def back_path
+    if claim.mentor_trainings.find_index(mentor_training).zero?
+      edit_claims_school_claim_mentor_path(claim.school, claim, mentor_training.mentor_id)
+    else
+      edit_claims_school_claim_mentor_training_path(
+        claim.school,
+        claim,
+        claim.mentor_trainings[claim.mentor_trainings.index(mentor_training) - 1].id,
+      )
+    end
+  end
+
+  def success_path
+    if claim.mentor_trainings_without_hours.any?
+      edit_claims_school_claim_mentor_training_path(
+        claim.school,
+        claim,
+        claim.mentor_trainings_without_hours.first.id,
+      )
+    else
+      check_claims_school_claim_path(claim.school, claim)
+    end
+  end
+
+  def custom_hours_checked?
+    custom_hours? || mentor_training.custom_hours_completed.present?
+  end
+
+  private
+
+  def custom_hours?
+    hours_completed == "on"
+  end
+
+  def updated_mentor_training
+    @updated_mentor_training ||= begin
+      mentor_training.hours_completed = custom_hours? ? nil : hours_completed
```

Comment:
Rather than doing the `custom_hours?` check twice. You could move this to and `if else`, so you only need to do the check one.